lang/de/report_rolessitemap.php

```diff
@@ -0,0 +1,33 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Strings
+ *
+ * @package    report_rolessitemap
+ * @copyright  2022 Andreas Schenkel
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+$string['pluginname'] = 'Rollenübersicht der Kursbereichen';
+$string['privacy:metadata'] = 'Der Bereicht speichert keine personenbezogenen Daten.';
+$string['rolessitemap:view'] = 'Bericht sehen';
+$string['isactive'] = 'Bericht aktivieren';
+$string['configisactive'] = 'Wenn aktiviert, dann können Nutzernde mit der Berechtigung rolessitemap:view den den Bericht anzeigen lassen.';
+$string['isactiveforsiteadmin'] = 'Aktivieren für Siteadmins';
+$string['configisactiveforsiteadmin'] = 'Wenn der Report für normale Nutzende NICHT aktiviert ist (isactive=false) aber für Siteadmins aufrufbar sein soll, so dann kann dies hier aktiviert werden.';
+$string['reportisinactiv_msg'] = 'Bereicht ist nicht aktiviert oder Berechtigung fehlt.';
+$string['header.page'] = 'Übersicht der zugewiesenen Rolle in Kursbereichen';
```

Comment:
Strings should be sorted alphabetically in general

Do not use "header.page" as key, this is confusing with object notation in mustache